app/views/admin/users/show.html.haml

```diff
@@ -23,21 +23,27 @@
     %h2.govuk-heading-m
       Linked Suppliers
 
-    %table.govuk-table
-      %thead.govuk-table__head
-        %tr.govuk-table__row
-          %th.govuk-table__header Name
-          %th.govuk-table__header Actions
-      %tbody.govuk-table__body
-        - @user.memberships.each do |membership|
+    - if ! @user.memberships.empty?
```

Comment:
As above, go direct to the source with `@user.suppliers`. Also, `@user.suppliers.any?` removes the need for the `!`.